python/lsst/ts/cbp/component.py

```diff
@@ -3,18 +3,26 @@
 import logging
 import asyncio
 import types
-import enum
 
 
-class Status(enum.Flag):
-    """The status for the encoders.
-    """
+class Status:
+    """The status for the encoders."""
```

Comment:
Actually its the encoder status, which is basically if the encoder have a problem or not.

Reply:
Please explain that in the doc string, including what the values mean.

Please consider a less generic name, such as `EncoderStatus`.

If the values are booleans then please initialize them to False, rather than 0.